packages/node_modules/@webex/plugin-meetings/test/unit/spec/member/index.js

```diff
@@ -0,0 +1,25 @@
+import sinon from 'sinon';
+import {assert} from '@webex/test-helper-chai';
+
+import MemberUtil from '@webex/plugin-meetings/src/member/util';
+import Member from '@webex/plugin-meetings/src/member';
+
+
+describe('member', () => {
+  const sandbox = sinon.createSandbox();
+
+  afterEach(() => {
+    sandbox.restore();
+  });
+
+  it('checks isHandRaised is called on initialisation', async () => {
```

Comment:
nitpick: This test isn't checking initialisation, it's checking the `processParticipant` method, would be good to update the description to match.